labs/lab-2-javascript-game/script.js

```diff
@@ -0,0 +1,48 @@
+var playerHealth = 40;
+var grantHealth = 10;
+var winCount = 0;
+
+function getDamage() {
+    return Math.floor(Math.random() * 5) + 1;
+}
+
+function startCombat() {
+
+  const playerDmg = getDamage();
+  const grantDmg = getDamage();
+
+   console.log("You took ", playerDmg, "damage!");
+   playerHealth = playerHealth - playerDmg;
+   console.log("You have health ", playerHealth, "left.");
+
+   console.log("Grant took ", grantDmg, "damage!");
+   grantHealth = grantHealth - grantDmg;
+   console.log("Grant has health ", grantHealth, "left.");
+
+}
+
+
+function startGame() {
+  var play = prompt("Would you like to play?");
+
+  if (play === "Yes") {
+    var playerName = prompt("What is your name?");
+    while (winCount < 3) {
```

Comment:
This works, but it might read more clearly if there were two, nested loops: one for "play until someone dies" and one for "kill repeatedly until someone wins".